### PR TITLE
Update adeo.ts Add LEXMAN Acces controler

### PR DESCRIPTION
**Pull request by RAULSALABIGAS**
Add support for ADEO/LEXMAN controller

### CODE REVIEW

**Comment by RAULSALABIGAS** [comment]
I don't know how to upload the image

SIN-4-1-22_LEX

File: src/devices/adeo.ts [hunk 1]
```diff
@@ -325,6 +325,23 @@ const definitions: Definition[] = [
...
+        description: 'ENKI LEXMAN Access Control',
+        extend: extend.switch(),
+        // The configure method below is needed to make the device reports on/off state changes
+        // when the device is controlled manually through the button on it.
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const ep = device.getEndpoint(1);
+            await reporting.bind(ep, coordinatorEndpoint, ['genOnOff']);
+            await reporting.onOff(ep);
+        },
+        endpoint: (device) => {
```
Comment (Koenkk) [inline src/devices/adeo.ts:341]: is this really needed? Can you try without it?

**Review by RAULSALABIGAS — COMMENTED** [review]
This part was not necessary

**Comment by RAULSALABIGAS** [comment]
I have modified the code. Thank you very much for your work

**Comment by Koenkk** [comment]
I was refering to the endpoint part, the configure part is probably needed, please add it back.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2151

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**